xyzspaces/datasets/__init__.py

```diff
@@ -23,6 +23,10 @@
 
 import requests
 
+from xyzspaces import Space
+
+microsoft_buildings_space_id = "R4QDHvd1"
```

Comment:
constants should be all caps as per pep8 style - https://www.python.org/dev/peps/pep-0008/#constants